cmd/explain.go

```diff
@@ -97,7 +99,11 @@ func getNetworkDetails(network *net.IPNet) *networkDetailsToDisplay {
 	// Obtain the total count of addresses in the network.
 	count := core.GetAddressCount(network)
 	// Format the count as a human-readable string and store it in the details struct.
-	details.Count = message.NewPrinter(language.English).Sprintf("%d", count)
+	if count.Cmp(big.NewInt(math.MaxInt64)) == 1 {
```

Comment:
Why are we doing this exactly? Is it not possible to print always as a string for consistency otherwise?

Reply:
I tried to preserve the earlier commits of keeping the comma's in the printed string (because it does look nice).
Eg. ```123,456``` vs. ```123456``` 
 NewPrinter does that, but it doesn't seem to interpret *big.Int as an interpret-able number so that's the quick fix to preserve comma printing for reasonably sized numbers. 
I could probably carve out a simple formatter, but I didn't know how much hassle it would be, so I opted to just preserve the earlier function first before trying to add in extra niceties.